src/server/auth.js

```diff
@@ -89,8 +89,10 @@ async function fetchEditors(token, query, page) {
   ).then(res => res.json());
 }
 
-export const getEditors = async managementToken => {
-  const query = 'include_totals=true&q=app_metadata.roles:"drafts:write"';
+export const getEditors = async (managementToken, roles) => {
+  // In the case of multiple roles we chose the one with "write" scope.

```

Comment:
litt usikker her, før så hadde vi hardkoda `drafts:write` her på alle requests, men nå som vi tillater flere roller, vet jeg ikke hva vi burde velge som default. setter write scope inntil videre

Reply:
Så vidt jeg kan se så har auth0 apiet `OR` og `AND` operatorer. Kan vi ikke bruke en av de? :)

Reply:
Kuult! Lettet etter nøyaktig dette, for å kunne se strukturen på spøringene osv. nice! Men spørsmålet nå er, skal vi da `OR`'e alle roles som kommer inn i funksjonen? eller hva slags roller er det vi skal lage spøringen med?

Reply:
Om du berre sender inn ei rolle til funksjonen fell heile dette problemet bort.